src/Components/bookHub/BookCard.js

```diff
@@ -50,31 +50,31 @@ export default function BookCard({
   useEffect(getAuthor, []);
```

Comment:
This throws errors if the author id is invalid, can you add a check for if the id is MISSING AUTHOR here or handle the case in the component above?